helpers/localizePrice.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const common = require('./lib/common.js');
+
+const factory = globals => {
+    return function(price, locale) {
+
+        if (!common.isObject(price) || !common.isString(price.currency) || isNaN(price.value)) {
+            // Return empty string if this does not appear to be a price object
+            return ''
+        }
+
+        if (!common.isString(locale) || locale.length < 2) {
+            // Valid browser language strings are at least two characters
+            // https://www.metamodpro.com/browser-language-codes
+            // If provided locale is less than two characters (or not a string),
+            // Default to 'en-US'
+            locale = 'en-US';
```

Comment:
An alternate approach here would be to wrap the actual render step in a try/catch and simply fall back to `en-US` if we get any input that JavaScript doesn't recognize as a locale, which might be more resilient.

In Chrome this appears to raise a `RangeError`:

![image](https://user-images.githubusercontent.com/8922457/60458004-143eb680-9c03-11e9-9f3e-3dca9ccd3504.png)

Interested in feedback.

Reply:
I think it is a pretty reasonable approach - try and fallback. Browser know what it supports and we should just rely on that.

Reply:
I just realized we can just fall back to `price.formatted` if anything fails - nice clean fallback.

Reply:
Updated to be even more resilient!